[2024-09-05T01:31:38.769+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-09-05T01:31:38.801+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_sensors_DBU.get_file_name scheduled__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-05T01:31:38.812+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_sensors_DBU.get_file_name scheduled__2024-09-04T00:00:00+00:00 [queued]>
[2024-09-05T01:31:38.812+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 3
[2024-09-05T01:31:38.839+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): get_file_name> on 2024-09-04 00:00:00+00:00
[2024-09-05T01:31:38.845+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=102) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-09-05T01:31:38.847+0000] {standard_task_runner.py:63} INFO - Started process 104 to run task
[2024-09-05T01:31:38.851+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'data_sensors_DBU', 'get_file_name', 'scheduled__2024-09-04T00:00:00+00:00', '--job-id', '16', '--raw', '--subdir', 'DAGS_FOLDER/dag_sensor_csv.py', '--cfg-path', '/tmp/tmp_doxvpf2']
[2024-09-05T01:31:38.853+0000] {standard_task_runner.py:91} INFO - Job 16: Subtask get_file_name
[2024-09-05T01:31:38.933+0000] {task_command.py:426} INFO - Running <TaskInstance: data_sensors_DBU.get_file_name scheduled__2024-09-04T00:00:00+00:00 [running]> on host 2374d61fd1eb
[2024-09-05T01:31:39.233+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='anderson.coder.space@gmail.com' AIRFLOW_CTX_DAG_OWNER='ander_o' AIRFLOW_CTX_DAG_ID='data_sensors_DBU' AIRFLOW_CTX_TASK_ID='get_file_name' AIRFLOW_CTX_EXECUTION_DATE='2024-09-04T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-09-04T00:00:00+00:00'
[2024-09-05T01:31:39.239+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-09-05T01:31:39.504+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-09-05T01:31:39.509+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-09-05T01:31:39.589+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_sensors_DBU, task_id=get_file_name, run_id=scheduled__2024-09-04T00:00:00+00:00, execution_date=20240904T000000, start_date=20240905T013138, end_date=20240905T013139
[2024-09-05T01:31:39.666+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-09-05T01:31:39.709+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-09-05T01:31:39.726+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
